app/controllers/two_factor_authentication/otp_verification_controller.rb

```diff
@@ -1,7 +1,9 @@
 module TwoFactorAuthentication
-  class OtpVerificationController < DeviseController
+  class OtpVerificationController < ApplicationController
     include TwoFactorAuthenticatable
 
+    skip_before_action :handle_two_factor_authentication
```

Comment:
this action is called in https://github.com/18F/identity-idp/blob/06201b79b83a03f1f3ee67effddbe64c47a04c69/app/controllers/concerns/two_factor_authenticatable.rb#L6 and defined in https://github.com/18F/identity-idp/blob/30457ab271b0fc963b84f0ecb7b3483bce96a86f/config/initializers/two_factor_auth_controller_override.rb#L9 where it is skipped for Devise controllers, so I needed to skip it manually when removing the Devise inheritance from these controllers.